mxcubeweb/core/components/sampleview.py

```diff
@@ -350,8 +350,17 @@ def rotate_to(self, sid):
             phi_value = round(float(cp.as_dict().get("phi", None)), 3)
             if phi_value:
                 try:
-                    HWR.beamline.diffractometer.centringPhi.set_value(phi_value)
+                    logging.getLogger("user_level_log").info(
+                        "Rotating to position %s", phi_value
+                    )
+                    if hasattr(HWR.beamline.diffractometer, "centringPhi"):
+                        HWR.beamline.diffractometer.centringPhi.set_value(phi_value)
+                    elif hasattr(HWR.beamline.diffractometer, "phi_motor_hwobj"):
+                        HWR.beamline.diffractometer.phi_motor_hwobj.set_value(phi_value)
```

Comment:
Maybe this to avoid the backwards-incompatible change where this would silently fail if there is no motor `phi_motor_hwobj` nor `centringPhi`:

```suggestion
                    if hasattr(HWR.beamline.diffractometer, "phi_motor_hwobj"):
                        HWR.beamline.diffractometer.phi_motor_hwobj.set_value(phi_value)
                    else:
                        HWR.beamline.diffractometer.centringPhi.set_value(phi_value)
```